packages/react/src/redux.ts

```diff
@@ -0,0 +1,93 @@
+// @flow
+import * as Sentry from '@sentry/browser';
```

Comment:
What if we'd use `@sentry/minimal` instead? Then it'd work for node and browser without any changes.

Reply:
If we do this would we need to move it out of the react package then, as the react package currently uses browser.